ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -354,43 +357,29 @@ const CoinButtons = ({
       },
       { excludeMetaMetricsId: false },
     );
-    switch (account.type) {
-      ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
-      case BtcAccountType.P2wpkh: {
-        try {
-          // FIXME: We switch the tab before starting the send flow (we
-          // faced some inconsistencies when changing it after).
-          await dispatch(setDefaultHomeActiveTabName('activity'));
-          await sendMultichainTransaction(
-            BITCOIN_WALLET_SNAP_ID,
-            account.id,
-            chainId as CaipChainId,
-          );
-        } catch {
-          // Restore the previous tab in case of any error (see FIXME comment above).
-          await dispatch(setDefaultHomeActiveTabName(currentActivityTabName));
-        }
 
-        break;
+    if (isEvmAccountType(account.type)) {
+      // Native Send flow
```

Comment:
We still this event:

```suggestion
      // Native Send flow
      trackEvent(
        {
          event: MetaMetricsEventName.NavSendButtonClicked,
          category: MetaMetricsEventCategory.Navigation,
          properties: {
            token_symbol: 'ETH',
            location: 'Home',
            text: 'Send',
            chain_id: chainId,
          },
        },
        { excludeMetaMetricsId: false },
      );
```